manager/integration/tests/test_migration.py

```diff
@@ -123,7 +123,64 @@ def test_migration_with_unscheduled_replica(clients, volume_name):  # NOQA
     11. Enable the scheduling for the node and wait for rebuilding complete.
     12. Validate initially written test data.
     """
-    return
+    # Step 1
+    client = get_random_client(clients)
+
+    # test_nodes[0] : node which running script
+    # test_nodes[1] : migrate target node
+    # test_nodes[2] : node to schedule on/off
+    test_nodes = list()
+    nodes = client.list_node()
+    for node in nodes:
+        if node.name == common.get_self_host_id():
+            test_nodes.append(node)
+    for node in nodes:
+        if node.name != common.get_self_host_id():
+            test_nodes.append(node)
```

Comment:
Are these two iterations on nodes for getting the Longhorn node first, and then other nodes in the list?

Reply:
Yes, currently I can not find a more efficient way to do that so I just use 2 loop to come up then I can clearly know every node's role in this test case

Reply:
I think you can make the test simpler. WDYT? 

- Use nodes from `get_hosts_for_migration_test` to do the migration testings
- Enable/disable scheduling of any of them
- Just use the local node for the first attachment and write the data

You don't need to prepare a list to store 3 nodes' info, but just reuse an existing function to get the migration nodes to start the testing as above.

Reply:
Yes this is more simpler, I updated code by using `get_hosts_for_migration_test` to get nodes  for review